nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -274,6 +282,9 @@ def setup_experiment(self, config):
         # Only profile from rank 0
         self.profile = config.get("profile", False) and self.rank == 0
 
+        # Register post-epoch hooks. To be used as `self.model.apply(post_epoch_hook)`
+        self.post_epoch_hooks = config.get("post_epoch_hooks", [])
```

Comment:
Please add docstring for all additional config parameters

Reply:
Where is this hook used?
I dont see it in this PR. Is there a second PR blocked by this one?

Reply:
This is helpful for dynamic sparsity - we need to prune and grow connections (i.e. weights) and the end of each epoch.